operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -27,7 +27,7 @@ Scenarios where this procedure is applicable:
     ```
 
 1. Collect the following information from the NCN:
-    1. Determine the component name (xname) of the NCN by referring to the HMN of the system's SHCD file, if it has not been determined yet.
+    1. (`ncn-m#`) Determine the component name (xname) of the NCN by referring to the HMN of the system's SHCD file, if it has not been determined yet.
```

Comment:
Does this procedure need to be run on a master NCN? Or would it work on any NCN?

Reply:
I did the prereq stuff for the `Add a Standard Rack Node` procedure. I'm not sure what pre-req stuff is required for the `Add NCN Data` procedure.

Reply:
I made two PRs with the remaining linting suggestions I have, along with the addition of the prereq section for that page:
https://github.com/Cray-HPE/docs-csm/pull/2075
https://github.com/Cray-HPE/docs-csm/pull/2076

I think if you pull those in, my concerns are all addressed.